Blockzilla/UIConstants.swift

```diff
@@ -218,7 +218,11 @@ struct UIConstants {
         static let urlBarToolsetOffset: CGFloat = 60
         static let urlBarIPadToolsetOffset: CGFloat = 110
         static let textLogoOffset: CGFloat = -10 - browserToolbarHeight / 2
+        static let textLogoOffsetSmallDevice: CGFloat = 10 - browserToolbarHeight / 4
         static let textLogoMargin: CGFloat = 44
+        static let tipDescriptionMargin: CGFloat = 12
+        static let tipViewHeight: CGFloat = 75
+        static let tipViewBottomOffset: CGFloat = -6
```

Comment:
I think it's better if we keep the constants non-negative and use it like this `offset(-UIConstants.layout.tipViewBottomOffset)`

Reply:
That sounds fine, I will add this change. Thank you!